site/nettick.xml

```diff
@@ -91,5 +91,16 @@ node rabbit@cordelia down: net_tick_timeout
     ].</pre>
       </p>
     </doc:section>
+    <doc:section name="net-tick-frequency">
+      <doc:heading>Management API</doc:heading>
```

Comment:
We don't really use the term "management API" much. Either "management UI" or "HTTP API". I suggest using the latter with a link to `/management.html`.